re.rizin.cutter.yml

```diff
@@ -0,0 +1,96 @@
+app-id: re.rizin.cutter
+command: cutter
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+separate-locales: false
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm14
+build-options:
+  prepend-path: /usr/lib/sdk/llvm14/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm14/lib
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --allow=devel
+
+add-extensions:
+  re.rizin.cutter.plugin:
+    version: stable
+    directory: extensions
+    add-ld-path: lib
+    merge-dirs: plugins;bin;lib
+    subdirectories: true
+    no-autodownload: true
+
+modules:
+  #stolen from https://github.com/flathub/org.freecadweb.FreeCAD/blob/master/org.freecadweb.FreeCAD.yaml
+  - name: pyside2
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TESTS=OFF
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-5.15.5-src/pyside-setup-opensource-src-5.15.5.tar.xz
+        sha256: 3920a4fb353300260c9bc46ff70f1fb975c5e7efa22e9d51222588928ce19b33
+      - type: shell
+        commands:
+          - mkdir -p /app/include/qt5tmp && cp -R /usr/include/Qt* /app/include/qt5tmp # https://bugreports.qt.io/browse/PYSIDE-787
+          - sed -i 's|\(--include-paths=\)|\1/app/include/qt5tmp:|' sources/pyside2/cmake/Macros/PySideModules.cmake
+
+  - name: rizin
+    buildsystem: meson
+    config-opts:
+      - --buildtype=release
+      - -Drizin_plugins=extensions/lib/rizin/plugins
+      - -Dinstall_sigdb=true
+      - -Dsubprojects_check=false # we manually add sigdb subproject
+    sources:
+      - type: archive
+        url: https://github.com/rizinorg/rizin/releases/download/v0.4.1/rizin-src-v0.4.1.tar.xz 
+        sha256: 669d956b997820a36e423cf35d482aa99849254b9658cef6844459912dfdbfb8
+      - type: git
+        url: https://github.com/rizinorg/sigdb
+        branch: main
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/flathub/flathub/pull/2251/commits/4249cea1c7be1c62c1847c2d3c773161079f9da1